s4e-web/src/app/views/map-view/view-manager/overlay-list-modal/wms-url.utils.ts

```diff
@@ -2,19 +2,19 @@ import { AbstractControl } from '@angular/forms';
 
 /* BASE URL */
 const HTTP_PREFIX = '(http(s?):\\/\\/)?';
-const LOCALHOST = '(localhost)';
-const DOMAIN_NAME = '((www\\.)?(([a-zA-Z0-9\\-\\_]*\\.)+)([a-zA-Z0-9\\-\\_]{2,3}))';
-const LOCAL_SUB_PAGES = '((\\/[a-zA-Z0-9]*)+)';
-const IP = '(\\b(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\\b)';
+const LOCALHOST = 'localhost';
+const DOMAIN_NAME = '(www\\.)?((\\.?)[a-zA-Z_\\.-]+)+(\\.[a-zA-Z0-9]{1,3})';
```

Comment:
Don't handle localhost separately, it should fit in the domain validation. A domain doesn't have to have a dot in it. You can have a valid domain defined in `/etc/hosts` such as `my-great-wms-server`.

Reply:
I agree